src/LedgerWallet/packages.config

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="hidlibrary" version="3.2.31.0" targetFramework="net45" />
+  <package id="Microsoft.NETCore.Platforms" version="1.1.0" targetFramework="net45" />
+  <package id="NStratis" version="4.0.0.77" targetFramework="net45" />
```

Comment:
This should reference the NBitcoin project in the solution instead

Reply:
Removed